src/jobs/megalinter.yml

```
@@ -146,29 +106,4 @@ steps:
-            token: <<parameters.github_token>>
-        - run:
-            environment:
-              GIT_USER_EMAIL: <<parameters.git_user_email>>
-              GIT_USER_NAME: <<parameters.git_user_name>>
-              GIT_COMMIT_MESSAGE: <<parameters.git_commit_message>>
-              GIT_PR_TITLE: <<parameters.git_pr_title>>
-              GIT_BRANCH_NAME: <<parameters.git_branch_name>>
-              GIT_BASE_BRANCH: <<parameters.git_base_branch>>
-            name: Set up Git
-            command: <<include(scripts/setup_git.sh)>>
+              - megalinter-reports
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)